modules/apps/foundation/roles/roles-admin-web/bnd.bnd

```diff
@@ -1,5 +1,6 @@
 Bundle-Name: Liferay Roles Admin Web
 Bundle-SymbolicName: com.liferay.roles.admin.web
 Bundle-Version: 1.0.10
+Export-Package: com.liferay.roles.admin.web.util
```

Comment:
Why is this package being exported?

It seems very tied to the particular implementation of the web layer in this module.